django_socio_grpc/tests/fakeapp/services/related_field_model_service.py

```diff
@@ -1,9 +1,14 @@
 from fakeapp.models import RelatedFieldModel
-from fakeapp.serializers import RelatedFieldModelSerializer
+from fakeapp.serializers import RelatedFieldModelSerializer, SimpleRelatedFieldModelSerializer
 
 from django_socio_grpc import generics
 
 
 class RelatedFieldModelService(generics.AsyncModelService):
     queryset = RelatedFieldModel.objects.all().order_by("uuid")
     serializer_class = RelatedFieldModelSerializer
+
+
+class SimpleRelatedFieldModelService(generics.AsyncModelService):
```

Comment:
miss comment to explain that it exist for the _ConvertScalarFieldValue functionnality